Tests/WUtilsTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  WUtilsTests.swift
+//  WMobileKit
+//
+//  Copyright 2016 Workiva Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+import Quick
+import Nimble
+@testable import WMobileKit
+
+class WUtilsTests: QuickSpec {
```

Comment:
If we do https://github.com/Workiva/w-mobile-kit/pull/132/files#r106484258 we'd change this.